<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgMakePlist</class>
 <widget class="QDialog" name="dlgMakePlist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dlgMakePlist</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rms_update.qrc">
    <normaloff>:/images/xml.png</normaloff>:/images/xml.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="openBtn">
       <property name="text">
        <string> Open select file </string>
       </property>
       <property name="icon">
        <iconset resource="rms_update.qrc">
         <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>  Attention: Please select the path of the program file.</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Path</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Md5</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Progress</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>X</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="uploadLabel">
       <property name="text">
        <string>Welcome to use update</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="makeBtn">
       <property name="text">
        <string>Make</string>
       </property>
       <property name="icon">
        <iconset resource="rms_update.qrc">
         <normaloff>:/images/xml.png</normaloff>:/images/xml.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="uploadBtn">
       <property name="text">
        <string>upload</string>
       </property>
       <property name="icon">
        <iconset resource="rms_update.qrc">
         <normaloff>:/images/upload.png</normaloff>:/images/upload.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitBtn">
       <property name="text">
        <string>Quit</string>
       </property>
       <property name="icon">
        <iconset resource="rms_update.qrc">
         <normaloff>:/images/quit.png</normaloff>:/images/quit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rms_update.qrc"/>
 </resources>
 <connections/>
</ui>
